docs/reference/runtime/_shared-configuration.md

```diff
@@ -79,6 +67,16 @@ runtime. Each application object supports the following settings:
 - **`packageManager`** (`string`) - The package manager to use when using the `install-dependencies` or the `resolve` commands of `wattpm-utils`. Default is to autodetect it, unless it is specified via command line.
 - **`preload`** (`string` or `array` of `string`s): A file or a list of files to load before the application code.
 - **`nodeOptions`** (`string`): The `NODE_OPTIONS` to apply to the application. These options are appended to any existing option.
+- **`permissions`** (`object`): Configure application-level security permissions to restrict file system access. Supported properties are:
+  - **`fs`**:
+    - **`read`** (`array` of `string`s): Array of file system paths the application is permitted to read from. Uses the same syntax as Node.js [--allow-fs-read](https://nodejs.org/dist/latest/docs/api/cli.html#--allow-fs-read).
+    - **`write`** (`array` of `string`s): Array of file system paths the application is permitted to write to. Uses the same syntax as Node.js [--allow-fs-write](https://nodejs.org/dist/latest/docs/api/cli.html#--allow-fs-write).
+
+  When filesystem permissions are enabled, certain paths are automatically added to maintain application functionality:
+  - The current Watt project's `node_modules` directory
+  - The application's own `node_modules` directory
+  - Any `node_modules` directories found in parent directories of the runtime path
```

Comment:
Please add the other limitations about child processes, worker threads, native modukle and link to the main permission system

Reply:
Done